tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -28,6 +28,9 @@
 from yamlformat.validator import entity_type_lib
 from yamlformat.validator import findings_lib
 from yamlformat.validator import presubmit_validate_types_lib as pvt
+import datetime
+
+# pylint: disable=consider-using-f-string
```

Comment:
I prefer .format whenever I'm configuring a big string because its usually easier to read.

For example:

```
print('[ERROR]\t{time}\tEntity {guid} ({code}) defines field '
      '{field} with raw value {raw_value} mapped to more than '
      'one state: {state} and {other}'
      .format(time=datetime.datetime.now(),
              guid=entity.guid,
              code=entity.code,
              field=qualified_field_name,
              raw_value=raw_value,
              state=state,
              other=ft.raw_values[raw_value])
      )
```

The same line in f-string:
```
print(f'[ERROR]\t{datetime.datetime.now()}\tEntity {entity.guid} '
      f'({entity.code}) defines field {qualified_field_name} with '
      f'raw value {raw_value} mapped to more than one state: '
      f'{state} and {ft.raw_values[raw_value]}.'
      )
```

While the f-string is shorter, I think the .format is easier to understand what the error is saying when reading the code; to understand the f-string you have to interpret what each input means and what its for, and for things like `datetime.datetime.now()` and `ft.raw_values[raw_value]` it seems cleaner to do it the first way. 

Why does g3 require fstring formatting?